<launch>
  <arg name="interface_name" default="wlan0"/>
  <arg name="rssi_sampling_rate" default="1" />
  <arg name="other_sampling_rate" default="1" />
  <arg name="node_name" default="node1" />

  <node pkg="network_analysis" type="wireless_quality.py" name="wireless_quality" output="screen">
    <param name="INTERFACE_NAME" value="$(arg interface_name)"/>
    <param name="update_rate_wireless_quality" value="$(arg rssi_sampling_rate)"/>
    <remap from="/network_analysis/wireless_quality" to="$(arg node_name)/network_analysis/wireless_quality" />
  </node>
</launch>
